patches/weston/0002-Add-xwayland-support-for-managarm.patch

```diff
@@ -1,30 +0,0 @@
-From 717a54091fe9e2b71d5e00c44b776d8c2c33b0b1 Mon Sep 17 00:00:00 2001
```

Comment:
Did you test X based apps without this patch? I'm pretty sure it remains needed and I'm going to double check that myself before we accidentally break Xwayland.

Reply:
Xwayland seems to work for me. I tested `xclock` and some command-line utilities (I don't remember right now) that operated correctly.